extensions/azure/azure-test/src/testFixtures/java/org/eclipse/dataspaceconnector/azure/testfixtures/AbstractAzureBlobTest.java

```diff
@@ -54,18 +54,23 @@ public void setupClient() {
     }
 
     @NotNull
-    private BlobServiceClient getBlobServiceClient(String accountName, String key) {
+    private static BlobServiceClient getBlobServiceClient(String accountName, String key) {
+        return getBlobServiceClient(accountName, key, getEndpoint(accountName));
+    }
+
+    @NotNull
+    public static BlobServiceClient getBlobServiceClient(String accountName, String key, String endpoint) {
```

Comment:
Since this is used by a class that's not in the `AbstractAzureBlobTest` hierarchy, makes me wonder if would be better to move it to an another class.

Reply:
Created `TestFunctions` class